src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/SQLIntegTestCase.java

```diff
@@ -103,6 +103,25 @@ protected Request getSqlRequest(String request, boolean explain) {
         return sqlRequest;
     }
 
+    protected String executeQuery(String query, String requestType) {
+        try {
+            String endpoint = "/_opendistro/_sql?format=" + requestType;
+            String requestBody = makeRequest(query);
+
+            Request sqlRequest = new Request("POST", endpoint);
+            sqlRequest.setJsonEntity(requestBody);
+
+            RestClient restClient = ESIntegTestCase.getRestClient();
+            Response response = restClient.performRequest(sqlRequest);
+            Assert.assertEquals(200, response.getStatusLine().getStatusCode());
+            String responseString = TestUtils.getResponseBody(response, true);
```

Comment:
Is `true` required? If not, all these 4 lines seem same as `executeRequest(request)`.

Reply:
It's easier to debug it this way (you don't need to do additional formatting) :)